src/Lachain.Core/RPC/HTTP/Web3/Web3DataFormatUtils.cs

```diff
@@ -202,13 +207,13 @@ public static IHashTrieNode NodeFromJson(JObject jsonNode)
                 ["gas"] = Web3Number(receipt.Transaction.GasLimit),
                 ["gasPrice"] = Web3Number(receipt.Transaction.GasPrice),
                 ["hash"] = Web3Data(receipt.Hash),
-                ["input"] = Web3Data(receipt.Transaction.Invocation),
+                ["input"] = Web3Number(receipt.Transaction.Invocation),
```

Comment:
Are you sure that this field is a number?  Invocation is a piece  of the bytecode really,  it can't be used as a number